sdks/python/build.gradle

```diff
@@ -285,18 +285,18 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
 //
 //  Docker (2a):
 //
-//    ./gradlew :beam-sdks-python:portableWordCount
+//    ./gradlew :beam-sdks-python:portableWordCount -PoutputDir=<Path to distributed file system>
 //
 //  Local JobService (2b):
 //
-//    ./gradlew :beam-sdks-python:portableWordCount -PjobEndpoint=localhost:8099
+//    ./gradlew :beam-sdks-python:portableWordCount -PoutputDir=<Path to distributed file system> -PjobEndpoint=localhost:8099
 //
 task portableWordCount(dependsOn: 'installGcpTest') {
   doLast {
     // TODO: Figure out GCS credentials and use real GCS input and output.
     def options = [
             "--input=/etc/profile",
-            "--output=/tmp/py-wordcount-direct",
+            "--output=${project.property('outputDir')}",
```

Comment:
Not sure about this change. This will prevent people from easily running the wordcount example. With `--streaming` and `--environment_cache_millis` it works fine.

I'd rather update the docs to educate about the local writes inside the folders.

Reply:
We don't support local filesystem with docker execution mode. This was actually failing with batch mode as the temp files were created inside docker.
We can also move to process based execution but it requires difficult artifact distribution.

Reply:
It used to work in batch and streaming. It would be nice to keep it that way for demoing purposes. Most people don't have a shared file system available.

Reply:
https://issues.apache.org/jira/browse/BEAM-5440 would be the best way to fix, mounting a local directory.